gdal/configure.ac

```diff
@@ -1463,7 +1463,23 @@ if test "$LIBZ_SETTING" != "no" ; then
   fi 
 fi
 
+dnl ---------------------------------------------------------------------------
+dnl LERC and MRF are coupled
```

Comment:
Hum, are they ?
Since GDAL 2.4.0, the GTiff driver can build a LERC TIFF plugin, and this is not related at all with MRF
And normally (didn't try recently though), the MRF driver should be able to build without LERC

Reply:
hmm, my bad, I was under the idea that lerc required mrf, but looking at the code that is clearly not the case, I'll revert the lerc part of this PR. FYI, mrf builds fine without lerc, and that was accounted for in the proposed change